test.js

```diff
@@ -34,29 +34,19 @@ const mochaCakes2Rules = {
   ],
 };
 
-const { env, rules, parserOptions } = require(".");
-
 module.exports = {
-  parserOptions,
+  env: { mocha: true },
+  globals: { ...mochaCakes2Globals },
+  plugins: [ "no-only-tests", "chai-friendly" ],
   rules: {
-    ...rules,
+    ...mochaCakes2Rules,
```

Comment:
is this backwards compatible, the original rules are not included?

Reply:
Yes, they were always inherited from the root config file, but overwritten with the same rules in the test config just. I have tested that the inheritance works as expected.